modules/rosa-parent-commands.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/manage-objects-with-rosa.adoc
+
+[id="rosa-parent-commands{context}"]
+= Parent commands
+
+The `rosa` CLI uses parent commands with child commands to manage objects. The parent commands are `create`, `edit`, `delete`, `list`, and `describe`. Not all parent commands can be used with all child commands. For more information, see the specific reference topics that describes the child commands.
+
+[id="rosa-create_{context}"]
+== create
+
+Creates an object or resource when paired with a child command.
+
+.Example
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster
+----
+
+[id="rosa-edit_{context}"]
+== edit
+
+Edits options for an object, such as making a cluster private.
+
+.Example
+[source,terminal]
+----
+$ rosa edit cluster mycluster --private
```

Comment:
I had suggested about not using -c so I see that you removed it. But just checking whether it needs to have `--cluster=mycluster` like the other commands do?

Reply:
Yes, it does. Thanks!